index.html

```diff
@@ -229,5 +229,20 @@ <h2 class="categorie-title">
                 </li>
             </ul>
         </section>
+        <section class="form" id="form-id">
+            <h1 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form action="https://formspree.io/f/xvongony" method="POST" class="form-input">
+                <label class="input-name">
+                    <input type="text" class="full-name"name="full-name" id="full-name" maxlength="30" placeholder="Full name" required>
+                </label>
+                <label class="input-email">
+                    <input type="email" class="email" name="email" id="email" placeholder="Email address" required>
+                </label>
+                <label class="input-text">
+                    <textarea name="text_area" id="text_area" maxlength="500" placeholder="Enter text here" required></textarea>
+                </label>
+                <button type="submit" class="submit-button">Get in touch</button>
+            </form>
+        </section>
```

Comment:
I think your project is lacking some features in the design, the background image, social links, and emails are currently not in your project, You probably may have forgotten to add them😉 kindly fix that🤗

the Figma design🤗
<img width="160" alt="your_form" src="https://user-images.githubusercontent.com/64643995/212990816-a7e6cda6-50e9-4a26-9acb-0e06018a4c21.png">